### PR TITLE
Fix `using` so resolution works on absolute descriptors

### PR DESCRIPTION
**Pull request by GavinMendelGleason**

### CODE REVIEW

File: src/core/query/resolve_query_resource.pl [hunk 1]
```diff
@@ -729,6 +730,11 @@
     ->  Filter = type_filter{ types : [instance,schema] }
     ;   Filter = type_name_filter{ type : Type }).
 
+
```
Comment (spl) [inline src/core/query/resolve_query_resource.pl:733]:
```suggestion
```

File: src/core/query/woql_compile.pl [hunk 2]
```diff
@@ -1069,15 +1069,17 @@
 
 assert_pre_flight_access(Context, _AST) :-
     is_super_user(Context.authorization, Context.prefixes),
-    % This probably makes all super user checks redundant.
+%   This probably makes all super user checks redundant.
```
Comment (spl) [inline src/core/query/woql_compile.pl:1072]:
```suggestion
    % This probably makes all super user checks redundant.
```

File: src/core/query/woql_compile.pl [hunk 3]
```diff
@@ -5309,3 +5311,42 @@
...
 
 :- end_tests(store_load_data).
+
+:- begin_tests(preflight).
+:- use_module(core(util/test_utils)).
+:- use_module(core(api)).
+:- use_module(core(query)).
+:- use_module(core(triple)).
+:- use_module(core(transaction)).
+%:- use_module(library(terminus_store)).
```
Comment (spl) [inline src/core/query/woql_compile.pl:5321]:
```suggestion
```

File: src/core/query/woql_compile.pl [hunk 4]
```diff
@@ -1072,12 +1072,14 @@
...
-                   Read,
-                   Write),
+    do_or_die(
+        find_resources(AST,
+                       (Context.default_collection),
+                       (Context.filter),
+                       (Context.write_graph),
+                       Read,
+                       Write),
+        error(find_resource_pre_flight_failure_for(AST))),
```
Comment (spl) [inline src/core/query/woql_compile.pl:1082]: Is this an error that we should catch and report? If so, we should update `api_error.pl` and give a meaningful error message.

File: src/core/query/woql_compile.pl [hunk 5]
```diff
@@ -5309,3 +5311,41 @@
...
 :- end_tests(store_load_data).
+
+:- begin_tests(preflight).
+:- use_module(core(util/test_utils)).
+:- use_module(core(api)).
+:- use_module(core(query)).
+:- use_module(core(triple)).
+:- use_module(core(transaction)).
+
+test(json_unbound_capture, [
```
Comment (spl) [inline src/core/query/woql_compile.pl:5322]: Is this related to the rest of the PR or an accidental inclusion?

**Review by spl — CHANGES_REQUESTED** [review]

File: src/core/query/woql_compile.pl (hunk 4 above)
Comment (GavinMendelGleason) [reply src/core/query/woql_compile.pl]: Added

File: src/core/query/woql_compile.pl (hunk 5 above)
Comment (GavinMendelGleason) [reply src/core/query/woql_compile.pl]: Fixed naming

**Review by spl — APPROVED** [review]

**Pull request merged**